src/turn.js

```diff
@@ -1,31 +1,64 @@
 class Turn {
-	constructor(players) {
-		this.players = players;
-		this.guess = [];
-		this.endTurn = false;
-	}
+  constructor(round) {
+    this.round = round;
+    this.player = round.getCurrentPlayer();
+    round.choosePuzzle();
+    this.puzzle = round.currentPuzzle[0];
+    this.wheel = round.wheelData;
+  }
 
-	spinWheel() {
+  spinWheel(consonant) {
+    // spin wheel on DOM, get wedge value
+    if (wheel.wedge === 'BANKRUPT') {
+      this.player.currentScore = 0;
```

Comment:
If the player is the class that holds onto its score value, the player class should be the only thing that can manipulate that value. e.g. we'd want to be calling something like `this.player.updateScore(0)` rather than manually assigning it to 0 here.